clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -56,12 +56,33 @@ trait IoUtil {
     gs.exists(path) == 0
   }
 
+  def getMd5HashOfGoogleObject(path: String): String = {
+    new GsUtil(None).md5Hash(path)
+  }
+
+  def getSizeOfGoogleObject(path: String): Long = {
+    new GsUtil(None).du(path).head.split(" +").head.toLong
+  }
+
+  def listGoogleObjects(path: String): Seq[String] = {
+    new GsUtil(None).ls(path)
+  }
+
+
 }
 object IoUtil extends IoUtil {}
 
 //we should consider moving this to api usage instead of gsutil
 class GsUtil(stateDir: Option[Path]) {
 
+  def ls(path: String): Seq[String] = {
+    runGsUtilAndGetStdout(Seq("ls", path)).split("\n")
+  }
+
+  def du(path: String): Seq[String] = {
+    runGsUtilAndGetStdout(Seq("du", path)).split("\n")
```

Comment:
You can make this always print one line by adding the "-s" flag.

Reply:
Since this is in GsUtil and not IoUtil, I'm leaving it as more general. There may be a time when we want to get the sizes of all files under an object in the cloud.